contributors/design-proposals/cloudprovider-storage-metrics.md

```diff
@@ -49,37 +49,46 @@ Since we are interested in count(or rate) and latency percentile metrics of API
 the external Cloud Provider - we will use [Histogram](https://prometheus.io/docs/practices/histograms/) type for
 emitting these metrics.
 
-We will be using `HistogramVec` type so as we can attach dimensions at runtime. Whenever available
-`namespace` will reported as a dimension with the metric.
+We will be using `HistogramVec` type so as we can attach dimensions at runtime.
 
 ### GCE Implementation
 
-For GCE we simply use `gensupport.RegisterHook()` to register a function which will be called
-when request is made and response returns.
-
 To begin with we will start emitting following metrics for GCE. Because these metrics are of type
-`Summary` - both count and latency will be automatically calculated.
-
-1. gce_instance_list
-2. gce_disk_insert
-3. gce_disk_delete
-4. gce_attach_disk
-5. gce_detach_disk
-6. gce_list_disk
-
+`Histogram` - both count and latency will be automatically calculated.
+
+All gce metrics will be named - `cloudprovider_gce_api_duration_seconds`. api request
+being made will be reported as dimensions. The cloudprovider maintainer may choose to add additional dimensions
+as needed.
+
+The complete list of time series will look like:
```

Comment:
Remove "complete"? This is not a complete list, it's a list of examples? It'll make the document hard to maintain if we have to keep it in sync with the code.